autogpt_platform/backend/backend/blocks/llm.py

```
@@ -318,7 +332,49 @@ def max_output_tokens(self) -> int | None:
     ),  # gpt-4o-2024-08-06
     LlmModel.GPT4_TURBO: ModelMetadata(
         "openai", 128000, 4096, "GPT-4 Turbo", "OpenAI", "OpenAI", 3
-    ),  # gpt-4-turbo-2024-04-09
+    ),
+    REGOLO_APERTUS_70B: ModelMetadata(
```

Ruff (0.15.12)

[error] Undefined name `REGOLO_APERTUS_70B`

(F821)